src/exceptions/LabellerException.class.php

```diff
@@ -1,16 +1,27 @@
 <?php
-require_once TRIPOD_DIR . '/exceptions/TripodException.class.php';
+
+namespace Tripod\Exceptions;
+
+require_once TRIPOD_DIR . '/exceptions/Exception.class.php';
 
 /**
  * @codeCoverageIgnore
  */
-class TripodLabellerException extends TripodException {
+class LabellerException extends Exception {
```

Comment:
This is just how PHP namespaces work.  If the class name is 'bare' (i.e. no namespace) it is in the current namespace (in this case ` Tripod\Exceptions `).